documentation/List-of-Action-plugins.md

```diff
@@ -538,24 +535,32 @@ Generates a PDF file based on a [Carbone](https://carbone.io/api-reference.html)
 | `appicationSerial`                       |                                            |
 | `templateId`                             |                                            |
 | `userId`                                 |                                            |
+| `data`                                   |                                            |
 | `additionalData`                         |                                            |
-| `description`                            |                                            |
 | `isOutputDoc`                            |                                            |
+| `description`                            |                                            |
 
 
 The Action utilises the internal `generatePDF` function, which is also accessible via the [`/generate-pdf` endpoint](API.md)
 
 `docTemplateId` specifies the uniqueId of the carbone template file (from the "file" table) and `options` optional can define a localisatiion to be used for dates and currency formatting.
 
-The data used by the action primarily comes from `applicationData` and `outputCumulative`, which are flattened/spread into a combined object to the carbone processer. However, if you wish to supply extra data, this can be added within an `additionalData` parameter. It gets sent to the carbone processer's "data" field like so:
+The data used by the action primarily comes from `applicationData` and `outputCumulative`, which are flattened/spread into a combined object to the carbone processer. Extra data (such as from individual responses, or other expressions) can be provided in one of two ways:
+
+- `data`: this parameter is an object in which you can define a simple mapping between field names required by Carbone and fields on `applicationData`. This is the same as the mapping available in `modifyRecord`, so please [see that plugin](#modify-record) info for more detail and examples.
```

Comment:
```suggestion
- `data`: this parameter is an object in which you can define a simple mapping between field names required by Carbone and fields on `applicationData`. This is the same as the mapping available in `modifyRecord`, so please [see modify-record plugin](#modify-record) info for more detail and examples.
```

Just better to have what is is instead of **that**